### PR TITLE
Extend development notes

### PR DESCRIPTION
**Pull request by blcham**
The goal of this PR is to provide more precise information on how to develop the tool.

### CODE REVIEW

**Comment by blcham** [comment]
FYI @LaChope

File: README.md [hunk 1]
```diff
@@ -13,51 +17,59 @@ npm run build
...
 
-If you experience issues on pre-commit using the GUI and that you are using a node version manager, such as nvm, add the following script:
+## Development
 
-```
-# ~/.config/husky/init.sh
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
+To develop the tool, the easiest way to run it is as follows:
+1. Run the complete tool as described in [Running complete tool in Docker](#running-complete-tool-in-docker)
```
Comment (LaChope) [inline README.md:38]: This is confusing to me, did you mean to configure the same way? IN this way, it sounds to me like you have to follow the same steps, which is not the case.

Comment (LaChope) [reply README.md:38]: Or you meant to do step 1 OR step 2?

Comment (blcham) [reply README.md:38]: Maybe what you are asking is described below at the line 45:
image

is it?

**Review by Kasmadei — COMMENTED** [review]
Looks good and useful for new people 👍

File: README.md (hunk 1 above)
Comment (LaChope) [reply README.md:38]: Yes, I meant this, I did not see it before so I think it's ok to leave it like that

**Pull request merged**